libnetwork/iptables/firewalld.go

```diff
@@ -12,16 +12,6 @@ import (
 	"github.com/pkg/errors"
 )
 
-// IPV defines the table string
-type IPV string
-
-const (
-	// Iptables point ipv4 table
-	Iptables IPV = "ipv4"
-	// IP6Tables point to ipv6 table
-	IP6Tables IPV = "ipv6"
-)
-
```

Comment:
Had a quick peek if I could find consumers of this. I did find one public import in https://github.com/KatharaFramework/NetworkPlugin/blob/8474abd27dcd2c31e90fbffa7d0561aa09549977/bridge/go-src/src/bridge_utils.go

If we want to be on the safe side, we could add aliases and deprecate those;

```go
// IPV defines the table string
//
// Deprecated: use [IPVersion]
type IPV = IPVersion

const (
	// Iptables point ipv4 table
	//
	// Deprecated: use [IPv4].
	Iptables IPV = IPv4
	// IP6Tables point to ipv6 table
	//
	// Deprecated: use [IPv6].
	IP6Tables IPV = IPv6
)
```